google/cloud/forseti/services/dao.py

```diff
@@ -322,12 +322,12 @@ class Resource(base):
         cai_resource_name = Column(String(4096))
         cai_resource_type = Column(String(512))
         full_name = Column(String(2048), nullable=False)
-        type_name = Column(get_string_by_dialect(dbengine.dialect.name, 512),
+        type_name = Column(get_string_by_dialect(dbengine.dialect.name, 700),
                            primary_key=True)
         parent_type_name = Column(
-            get_string_by_dialect(dbengine.dialect.name, 512),
+            get_string_by_dialect(dbengine.dialect.name, 700),
             ForeignKey('{}.type_name'.format(resources_tablename)))
-        name = Column(String(256), nullable=False)
+        name = Column(String(1024), nullable=False)
```

Comment:
If i remembered correctly, type_name is '{RESOURCE_TYPE}/{RESOURCE_NAME}', if type_name is 700 max then name cannot exceed that size

Reply:
Noted, thanks!